docs/source/reference/cli.rst

```diff
@@ -97,8 +97,8 @@ Storage CLI
    :prog: sky storage delete
    :nested: full
 
-Utils: ``show-gpus``, ``check``
----------------------------------------
+Utils: ``show-gpus``, ``check``, ``cost-report``
```

Comment:
Nit: A space shows up before the commas, any way to remove it? 
<img width="191" alt="image" src="https://github.com/skypilot-org/skypilot/assets/4416605/5d1a3e97-9f6a-4127-a972-7e58c6cc2487">

Reply:
Reworded.